pkg/network/event_common.go

```diff
@@ -246,6 +246,12 @@ func (c ConnectionStats) ByteKey(buf []byte) ([]byte, error) {
 	return buf[:n], nil
 }
 
+// IsShortLived returns true when a connection went through its whole lifecycle
+// between two connection checks
+func (c ConnectionStats) IsShortLived() bool {
+	return c.MonotonicTCPEstablished >= 1 && c.MonotonicTCPClosed >= 1
```

Comment:
Thinking about this some more, isn't this true for all closed TCP connections, regardless of when they started?

Reply:
Oh, you're right. I should actually change this to `c.LatTCPEstablished >= 1 && c.LastTCPClosed >= 1`